### DISCUSSION TITLE
resend MQTT discovery command from RPi/RA

### DISCUSSION THREAD

**Discussion by Menethoran** [mKeRix/room-assistant#403] [Q&A]
hey guys... 
How do we resend the MQTT discovery command for an MQTT broker to pick up a new device (or deleted device... in my case)?

**Comment by Menethoran** [comment]
Disregard...
It's not "not sending the discovery" 
It's just not sending the motion sensing integration... and im not sure why...
HA/MQTT discovered it (motion sensing) when first set up... but deleting the entity causes it never to be able to be added again (or, not to my knowledge so far)... unsure if it's an RA issue or an HA issue...

Help would be appreciated if i could be guided where to look...

**Reply by mKeRix** [reply]
If you just deleted it in Home Assistant you can usually get it to be re-discovered by restarting Home Assistant. You can check which entities room-assistant registers by running it in verbose mode, looking at the entities endpoint of the HTTP API or looking at the MQTT topics.

**Reply by Menethoran** [reply]
The raspberry pis repopulate in HA, but not the motion sensor portions for some reason (the binary.sensors that represent them)

**Comment by Menethoran** [comment]
How do you run RA in verbose mode (RA is running as an add-on in HA)
